packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -208,9 +223,16 @@ const NavigationMenuProvider: React.FC<NavigationMenuProviderProps> = (
     onChange: onValueChange,
     defaultProp: defaultValue,
   });
+  const [shouldSkipDelay, setShouldSkipDelay] = React.useState(false);
```

Comment:
It would be good if we could follow what was done in `Tooltip` for consistency purpose.
We track rather whether open should be delayed or not with then a separate `handleOpen` and `handleDelayedOpen`.
If we could stay as close as possible to this implementation that would be great 🙏